### DISCUSSION TITLE
Is MAUI so big? A "Hello World" program is 163M in size

### DISCUSSION THREAD

**Discussion by weiyoucheng** [dotnet/maui#11301] [Q&A]
Why is the MAUI application so large? A simple "Hello World" program has a volume of 163M, which is too large. I think there are many DLLs in the debug directory that are not available. Is there any other way to reduce the volume
![}8L)}J$QFI4`VQ~Z (Y8@XV](https://user-images.githubusercontent.com/112521929/201448145-50d73f5e-1561-4e94-9524-d219ab6b56a1.png)

**Comment by xtuzy** [comment]
related Infor: https://github.com/microsoft/WindowsAppSDK/issues/2478

**Comment by akhanalcs** [comment]
Please create Issue of this so it helps the team prioritize it.
